x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/flyout_body_no_top_padding.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import styled from 'styled-components';
+import { EuiFlyoutBody } from '@elastic/eui';
+
+/**
+ * Removes the `padding-top` from the `EuiFlyoutBody` component. Normally done when there is a
+ * sub-header present above the flyout body.
+ */
+export const FlyoutBodyNoTopPadding = styled(EuiFlyoutBody)`
```

Comment:
Should we move this to a common folder?

Reply:
Right now its only being used in Endpoint details (that I know of), but yes, if we need something similar in the future in another page, we would promote it up.